roles/matrix-bot-maubot/templates/systemd/matrix-maubot.service.j2

```diff
@@ -0,0 +1,40 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Maubot
+{% for service in matrix_bot_maubot_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_bot_maubot_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-maubot 2>/dev/null || true'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-maubot 2>/dev/null || true'
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-maubot \
+			--log-driver=none \
+            -e UID={{ matrix_user_uid }} \
+            -e GID={{ matrix_user_gid }} \
+            -v {{ matrix_bot_maubot_data_path }}:{{ matrix_bot_maubot_container_data_dir }}:z \
+			{% for arg in matrix_bot_maubot_container_extra_arguments %}
+			{{ arg }} \
+			{% endfor %}
+			--network={{ matrix_docker_network }} \
```

Comment:
Indentation seems a bit off compared to the other lines.

Reply:
![image](https://user-images.githubusercontent.com/388669/175761828-c9092119-9777-448f-a475-754841672df1.png)

Here's what I meant. Some of these lines are indented with spaces, others with tabs. In some editors, this may look the same, but on the diff, you can clearly see that it's not.

We generally prefer tabs.

Reply:
Ah did not see that in my editor 😅